src/vs/platform/actions/common/actions.ts

```diff
@@ -61,6 +61,17 @@ export class MenuId {
 	static readonly ViewItemContext = new MenuId();
 	static readonly TouchBarContext = new MenuId();
 	static readonly SearchContext = new MenuId();
+	static readonly MenubarFileMenu = new MenuId();
```

Comment:
@sbatten what about the other sub-menus that we need to support, like "Editor Layout" (which was just added) and the ones under the "Go" menu? I think we should make sure that they all are covered as well. I would probably also add a TODO to keep in mind that a better solution would be to implement them via some real sub-menu support in the menus.